background.js

```diff
@@ -71,13 +72,20 @@ function bufferData(tabId, message) {
   }
 
   if (message.event === 'scope:new') {
-    tabData.scopes[message.data.child] = {
```

Comment:
I note that this change no longer modifies the `tabData.scope` object. Is that on purpose?